GliaWidgets/Resources/en.lproj/Localizable.strings

```diff
@@ -197,8 +197,16 @@
 "call.accessibility.buttons.minimize.badgeValue.singleItem" = "";
 "call.accessibility.buttons.minimize.badgeValue.multipleItems" = "";
 "call.accessibility.buttons.minimize.titleAndBadgeValue" = "{buttonTitle}, {badgeValue}";
+
 "survey.action.cancel" = "Cancel";
 "survey.action.submit" = "Submit";
 "survey.action.yes" = "Yes";
 "survey.action.no" = "No";
 "survey.action.validationError" = "Please provide an answer.";
+
+"survey.accessibility.footer.cancelButton.label" = "Cancel";
+"survey.accessibility.footer.submitButton.label" = "Submit";
+"survey.accessibility.question.title.value" = "Required";
+"survey.accessibility.question.optionButton.selected.label" = "Selected: %@";
```

Comment:
I think to makes sense to follow existing approach with string replacement as on line 199 `{buttonTitle}, ...` for consistency with other localization.